### PR TITLE
Added ascii printing for symbols

### LINKED ISSUE #12158: Pretty printing doesn't print alphabetical subscripts properly
```
moorepants@garuda:src(master)$ ipython
Python 3.5.2 |Continuum Analytics, Inc.| (default, Jul  2 2016, 17:53:06) 
Type "copyright", "credits" or "license" for more information.

IPython 5.1.0 -- An enhanced Interactive Python.
?         -> Introduction and overview of IPython's features.
%quickref -> Quick reference.
help      -> Python's own help system.
object?   -> Details about 'object', use 'object??' for extra details.

In [1]: from sympy import symbols, init_printing

In [2]: init_printing()

In [3]: mW = symbols('m_W')

In [4]: mW
Out[4]: m_W
```

### PR DESCRIPTION
**Pull request by Subhash-Saurabh**
#### References to other Issues or PRs

Fixes #12158 

#### Brief description of what is fixed or changed

Added ascii printing for symbols since unicode does not have definitions for all symbols as subscripts 

```
>>> sym1 = Symbol('x_2__i')
>>> pprint(sym1, use_unicode=False)
 i
x 
 2
```
#### Release Notes

* printing
   * proper super and subscripts for ASCII-printing of symbols

### CODE REVIEW

File: sympy/printing/pretty/pretty.py [hunk 1]
```diff
@@ -74,8 +74,22 @@ def _print_atan2(self, e):
         return pform
 
     def _print_Symbol(self, e):
-        symb = pretty_symbol(e.name)
-        return prettyForm(symb)
+        def ascii_symbol(e):
```
Comment (asmeurer) [inline sympy/printing/pretty/pretty.py]: This logic should go in the pretty_symbol function.

**Comment by oscargus** [comment]
Conflict is just about imports. Move code to the pretty_symbol function as per the review comment.

**Comment by iamabhishek0** [comment]
Hey, @asmeurer is this issue still open because I would like to work on it.

**Comment by czgdp1807** [comment]
Closing this in favour of https://github.com/sympy/sympy/pull/18111
Thanks for the contributions. If you think this can be re-opened, please do so or just comment below.

**Pull request closed without merging**